app/models/peep.rb

```diff
@@ -0,0 +1,12 @@
+require 'dm-timestamps'
+
+class Peep
+  include DataMapper::Resource
+
+  property :id, Serial
+  property :text, Text, :required => true, :lazy => false
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)